src/router/storeSync/LayerParamConfig.class.js

```diff
@@ -110,16 +109,74 @@ function dispatchLayersFromUrlIntoStore(store, urlParamValue) {
                 )
             }
             log.debug(`  Add layer ${parsedLayer.id} to active layers`, layerObject)
+            if (layerObject.customAttributes?.features) {
+                layerObject.customAttributes.features
+                    .toString()
+                    .split(':')
+                    .forEach((featureId) => {
+                        featuresRequests.push(
+                            getFeature(
+                                store.getters.getLayerConfigById(parsedLayer.id),
+                                featureId,
+                                store.state.position.projection,
+                                store.state.i18n.lang
+                            )
+                        )
+                    })
+            }
         }
         return layerObject
     })
+
     promisesForAllDispatch.push(
         store.dispatch('setLayers', { layers: layers, dispatcher: STORE_DISPATCHER_ROUTER_PLUGIN })
     )
+    if (featuresRequests.length > 0) {
+        getAndDispatchFeatures(featuresRequests, store) // all features after promise
+    }
 
     return Promise.all(promisesForAllDispatch)
 }
 
+async function getAndDispatchFeatures(featuresPromise, store) {
+    const features = []
+    try {
+        const featuresReceived = await Promise.all(featuresPromise)
```

Comment:
We should maybe be more resilient to bad inputs here, and have something like `Promise.allSettled` (see [doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled) ) which should help ignore bad IDs that user could give use, not breaking the whole loading process